public/app/features/integrated-alerting/IntegratedAlertingPage.tsx

```diff
@@ -1,52 +1,81 @@
-import React, { FC, useMemo, useState } from 'react';
+import React, { FC, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
 import { TabsBar, TabContent, Tab, useStyles } from '@grafana/ui';
+import { UrlQueryValue } from '@grafana/data';
+import { getLocationSrv } from '@grafana/runtime';
+import { Breadcrumb, PageModel } from 'app/core/components/Breadcrumb';
+import { StoreState } from 'app/types';
 import { Messages } from './IntegratedAlerting.messages';
 import { getStyles } from './IntegratedAlerting.styles';
 import { TabKeys } from './IntegratedAlerting.types';
 import { Alerts, AlertRuleTemplate, AlertRules, NotificationChannel } from './components';
+import { DEFAULT_TAB } from './IntegratedAlerting.constants';
+
+const tabs = [
+  {
+    title: Messages.tabs.alerts,
+    id: TabKeys.alerts,
+    path: `integrated-alerting/${TabKeys.alerts}`,
+    component: <Alerts key={TabKeys.alerts} />,
+  },
+  {
+    title: Messages.tabs.alertRules,
+    id: TabKeys.alertRules,
+    path: `integrated-alerting/${TabKeys.alertRules}`,
+    component: <AlertRules key={TabKeys.alertRules} />,
+  },
+  {
+    title: Messages.tabs.alertRuleTemplates,
+    id: TabKeys.alertRuleTemplates,
+    path: `integrated-alerting/${TabKeys.alertRuleTemplates}`,
+    component: <AlertRuleTemplate key={TabKeys.alertRuleTemplates} />,
+  },
+  {
+    title: Messages.tabs.notificationChannels,
+    id: TabKeys.notificationChannels,
+    path: `integrated-alerting/${TabKeys.notificationChannels}`,
+    component: <NotificationChannel key={TabKeys.notificationChannels} />,
+  },
+];
+
+const pageModel: PageModel = {
+  title: 'Integrated Alerting',
+  path: 'integrated-alerting',
+  id: 'integrated-alerting',
+  children: tabs.map(({ title, id, path }) => ({ title, id, path })),
```

Comment:
nit: would be good to have this logic separated (e.g in utils file in the breadcrumb component). It might be useful in other places to map tabs to page model children.